server/controllers/events/events_controller_test.go

```diff
@@ -618,6 +618,73 @@ func TestPost_AzureDevopsPullRequestDeletedCommentIgnoreEvent(t *testing.T) {
 	})
 }
 
+func TestPost_AzureDevopsPullRequestCommentPassingIgnores(t *testing.T) {
+	u := "user"
+	user := []byte(u)
+
+	t.Log("when the event should not be ignored it should succeed until parsing section")
+	RegisterMockTestingT(t)
+	v := mocks.NewMockAzureDevopsRequestValidator()
+	p := emocks.NewMockEventParsing()
+	cp := emocks.NewMockCommentParsing()
+	cr := emocks.NewMockCommandRunner()
+	c := emocks.NewMockPullCleaner()
+	vcsmock := vcsmocks.NewMockClient()
+	repoAllowlistChecker, err := events.NewRepoAllowlistChecker("*")
+	Ok(t, err)
+	logger := logging.NewNoopLogger(t)
+	scope, _, _ := metrics.NewLoggingScope(logger, "null")
+	e := events_controllers.VCSEventsController{
+		TestingMode:                     true,
+		Logger:                          logger,
+		Scope:                           scope,
+		ApplyDisabled:                   false,
+		AzureDevopsWebhookBasicUser:     user,
+		AzureDevopsWebhookBasicPassword: secret,
+		AzureDevopsRequestValidator:     v,
+		Parser:                          p,
+		CommentParser:                   cp,
+		CommandRunner:                   cr,
+		PullCleaner:                     c,
+		SupportedVCSHosts:               []models.VCSHostType{models.AzureDevops},
+		RepoAllowlistChecker:            repoAllowlistChecker,
+		VCSClient:                       vcsmock,
+	}
```

Comment:
I've started this but I think it might require a bit more of a review (/is a little out of scope for this issue) so I'll create a separate PR for this